deploy.py

```diff
@@ -0,0 +1,129 @@
+#! /usr/bin/python
+
+import json
+import os
+import sys
+import subprocess
+
+module_json = 'module.json'
+version_key = 'version'
+
+def main():
+    '''
+    This script updates versions, builds and deploys new versions of the kubos-cli.
+
+    How to use this script:
+
+    To explicitly set the version of the new release, update the version to what you
+    want in the module.json file and commit and push those changes to master. Once
+    the tests pass this script will tag the repo at that new version number (that you set
+    in module.json), build and upload the module to pypi.
+
+    If you don't care about the version number, just merge your changes to master
+    of the Kubos-cli repo. Once the tests pass, this will bump the build # of the version,
```

Comment:
Nit: Formatting. Either use 'kubos-cli' since that's the name of the repo, or 'Kubos CLI' since that's the name of the product